pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -566,6 +566,13 @@ class ModelResponse:
     kind: Literal['response'] = 'response'
     """Message type identifier, this is available on all parts as a discriminator."""
 
+    vendor_metadata: dict[str, Any] | None = None
```

Comment:
Let's do this so this doesn't get included automatically in the representation -- that way we can revert the changes to `docs/agents.md`, `pydantic_ai_slim/pydantic_ai/agent.py`, and `tests/test_agent.py`;

```suggestion
    vendor_metadata: dict[str, Any] | None = field(default=None, repr=False)
```

Reply:
I get why we called this `metadata`, but I'm thinking `vendor_details` may be better as it's consistent with `Usage.details`. What do you think?

Reply:
I agree with you, done!